server/middleware/authorisationMiddleware.js

```diff
@@ -0,0 +1,11 @@
+const jwtDecode = require('jwt-decode')
+
+module.exports = (req, res, next) => {
+  if (res.locals && res.locals.user && res.locals.user.token) {
```

Comment:
Do you need to check for `**res.locals.user**` here if you're checking for `**res.locals.user**.token`?

Reply:
This is based on something I nabbed from the cat tool 😄 

I would have thought so though, if there's no user present on locals - then wouldn't it blow up with `no field token on undefined`?